modules/collabora_online_group/collabora_online_group.install

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Drupal\views\Entity\View;
+
+/**
+ * Implements hook_install().
+ */
+function collabora_online_group_install(): void {
+  if (
+    \Drupal::moduleHandler()->moduleExists('views') &&
+    $view = View::load('group_media')
+  ) {
```

Comment:
I notice we are using "early return" in the inner condition below, but not in this first condition.
One reason for this could be to allow us to add more steps later, which might be unrelated to views.
But instead, if we add other install steps in the future, we can just split all of them to other functions.
So, let's do the early return?
```suggestion
  if (
    !\Drupal::moduleHandler()->moduleExists('views') ||
    !($view = View::load('group_media'))
  ) {
    return;
  }
```

Reply:
Changed